packages/lesswrong/server/repos/TagsRepo.ts

```diff
@@ -248,15 +248,26 @@ class TagsRepo extends AbstractRepo<"Tags"> {
       -- TagsRepo.getTagsByParentTagId
       SELECT
         t_child.*,
-        COUNT(*) OVER() AS "totalCount"
+        COUNT(*) OVER() AS "totalCount",
+        GREATEST(
+          t_child."baseScore",
+          COALESCE(md."maxBaseScore", 0)
+        ) AS "maxScore"
```

Comment:
I think you don't need to pull this into the SELECT; you can just put the `GREATEST(...)` expression into the `ORDER BY` directly (and then not need to `delete` it from the returned rows later).